packages/aws-otel-collector/0001-change-logger-and-extraconfig-file-paths.patch

```diff
@@ -0,0 +1,40 @@
+From bc1879fd049490bd9907b9590b133f2d558589d0 Mon Sep 17 00:00:00 2001
+From: Kush Upadhyay <kushupad@amazon.com>
+Date: Thu, 15 Aug 2024 09:12:55 +0000
+Subject: [PATCH] change logger and extraconfig file paths
+
+Signed-off-by: Kush Upadhyay <kushupad@amazon.com>
+---
+ pkg/extraconfig/extraconfig.go | 2 +-
+ pkg/logger/logger.go           | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/pkg/extraconfig/extraconfig.go b/pkg/extraconfig/extraconfig.go
+index 9c14d5f7..c962b562 100644
+--- a/pkg/extraconfig/extraconfig.go
++++ b/pkg/extraconfig/extraconfig.go
+@@ -23,7 +23,7 @@ import (
+ )
+ 
+ var (
+-	UnixExtraConfigPath    = "/opt/aws/aws-otel-collector/etc/extracfg.txt"
```

Comment:
This config should still be under `/etc/aws-otel-collector`.

Reply:
Ack, will put the extraconfig file under the `/etc/aws-otel-collector` directory